website/docs/r/autoscale_setting.html.markdown

```diff
@@ -0,0 +1,826 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_autoscale_setting"
+sidebar_current: "docs-azurerm-resource-autoscale-setting"
+description: |-
+  Create an autoscale setting that can be applied to Virtual Machine Scale Sets, Cloud Services and App Service - Web Apps.
```

Comment:
I might be wrong, but aren't Cloud Services part of ASM? In which case - do they belong in this list?

_minor:_ can we match the convention seen in the CosmosDB Resource by updating `App Service - Web Apps` to `App Service (formally "Web Apps")`